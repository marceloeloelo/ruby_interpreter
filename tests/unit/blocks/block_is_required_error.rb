foo = [1,2,3,nil]

foo.each()
