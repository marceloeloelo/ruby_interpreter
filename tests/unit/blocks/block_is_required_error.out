BlockError: a block is required for method `each'
